tests/e2e/restart_test.go

```diff
@@ -84,7 +84,7 @@ func (s *RestartTestData) TestVolumeDetached(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// 4. Verify detached status. Volume should be detached (within the timeout)
-	status = verification.VerifyDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
+	status = verification.PollDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
```

Comment:
> https://github.com/vmware/docker-volume-vsphere/pull/1552#discussion_r126505314

@govint / @pshahzeb correct and valid point. We came across many instances during our initial phase with vDVS where `docker inspect` and esxcli was giving different issue hence the extensive verification. I